src/rules/extensions.js

```diff
@@ -164,8 +164,8 @@ module.exports = {
       ) || isScoped(importPath);
 
       if (!extension || !importPath.endsWith(`.${extension}`)) {
-        // ignore type-only imports
-        if (node.importKind === 'type') return;
+        // ignore type-only imports and exports
+        if (node.importKind === 'type' || node.importKind === 'type') return;
```

Comment:
These are the same condition.

Could you add a test that fails without this suggestion:
```suggestion
        if (node.importKind === 'type' || node.exportKind === 'type') return;
```

Reply:
There were already tests, but I screwed up the conditional here. Thanks for pointing that out!

Is it ok now?

Reply:
I didn't check; did the tests fail with the typo? if so, then it's good now :-) (altho there's still a bunch of failing tests)